_includes/landing-header.html

```diff
@@ -3,7 +3,7 @@
         <div class="cell small-12">
             <h3 class="logo"><a href="{{ "/" | relative_url }}">Extension Workshop</a></h3>
             <nav class="site-nav">
-                <a href="https://addons.mozilla.org/developers/addons" class="button">Register or Log in</a>
+                <a href="https://addons.mozilla.org/developers/addons" class="button">Manage&#47;Submit Extension</a>
```

Comment:
You can use a literal `/` here rather than an entity.

Reply:
👍